docs/cri/crictl.md

```diff
@@ -59,29 +59,28 @@ command. With the load command you inject a container image into the container
 runtime from a file. First you need to create a container image tarball. For
 example to create an image tarball for a pause container using Docker:
 ```console
-$ docker pull k8s.gcr.io/pause-amd64:3.2
-  3.2: Pulling from pause-amd64
+$ docker pull k8s.gcr.io/pause:3.4.1
```

Comment:
Since The `pause:3.4.1` image is a manifest list with multiple architecture, use `k8s.gcr.io/pause:3.4.1`

https://github.com/kubernetes/k8s.io/issues/1619#issuecomment-774590258